gadgets/Makefile

```diff
@@ -99,12 +100,9 @@ test-ig-local: build
 	go test -exec 'sudo -E' -v ./...
 
 .PHONY:
-test-k8s: build
-	IG_PATH=$(KUBECTL_GADGET) \
-	IG_RUNTIME=kubernetes \
-	CGO_ENABLED=0 \
-	IG_EXPERIMENTAL=true \
-	GADGET_REPOSITORY=$(GADGET_REPOSITORY) \
-	GADGET_TAG=$(GADGET_TAG) \
-	IG_FLAGS=$(IG_FLAGS) \
-	go test -exec 'sudo -E' -v ./...
+test-local: IG_PATH=$(IG)
```

Comment:
Not your changes, but a question: Why do we have `IG_PATH` and `IG` env variables in this Makefile? Can't we just have one?

Reply:
`IG_PATH` is the path to the binary executed for the test. `IG` is the default location of the ig binary, and `KUBECTL-GADGET` is the default location of the kubectl-gadget binary. Probably we can remove the IG and KUBECTL-GADGET ones.